### ISSUE TITLE
Make a Forge version.

### ISSUE THREAD

**Issue by EnderBro123** [qouteall/ImmersivePortalsMod#14]
Title says it all.

It shouldn't be that hard since this is such a small mod.

Please and thank you.

**Comment by iiThought** [comment]
PLeAsE wE neEd ThiS fOR a NeW ImpaCt

**Comment by Pyrofab** [comment]
I suggest you look at the source before saying this is a small mod. This mod requires in fact more code than most of the things you have in your modpack, and it makes fundamental changes to the game that are very heavily discouraged on Forge.
Porting this to Forge would require a massive investment of time and effort, and I can say this as I tried to port [the existing Forge equivalent](https://www.curseforge.com/minecraft/mc-mods/betterportals)) to Fabric. Took me 3 full-time days to port a tenth of the things required (to be fair, it was also a 1.12->1.14 port).

TLDR; be nice, don't assume anything about the effort required to code.

**Comment by qouteall** [comment]
Currently, forge does not support mixin. I may port this to forge when forge supports mixin.
